lib/tests/run_tests.js

```diff
@@ -123,6 +124,23 @@ module.exports = {
             eachCb();
           });
         }, next);
+      },
+      function runCoverage(next) {
+        if (!options.coverage) {
+          return next();
+        }
+
+        runCmd(`${fs.embarkPath('node_modules/.bin/istanbul')} report --root .embark --format html`,
+          {silent: false, exitOnError: false}, (err) => {
+            if (err) {
+              return next(err);
+            }
+            console.log(`Coverage report created. You can find it here: ${fs.dappPath('coverage/index.html')}\n`);
+            const opn = require('opn');
+            opn(fs.dappPath('coverage/index.html'));
+
+            setTimeout(next, 1000); // Slight delay to enable the browser to open. opn's promise calls too fast
```

Comment:
why not calling next in the result of the `opn` promise?

Reply:
Like I said in the comment, the promise returned immediately, without waiting for the open.